SPECS/kernel/config

```diff
@@ -293,11 +293,8 @@ CONFIG_TRACEPOINTS=y
 #
 CONFIG_CRASH_CORE=y
 CONFIG_KEXEC_CORE=y
-CONFIG_HAVE_IMA_KEXEC=y
-CONFIG_KEXEC=y
-CONFIG_KEXEC_FILE=y
-# CONFIG_KEXEC_SIG is not set
-# CONFIG_KEXEC_JUMP is not set
+# CONFIG_KEXEC is not set
+# CONFIG_KEXEC_FILE is not set
```

Comment:
I don't think we should disable `CONFIG_KEXEC_FILE`. I would be fine with disabling `CONFIG_KEXEC`, however, since it cannot and will not ever support signature verification.

Reply:
@allenpais ping on this. `CONFIG_KEXEC_FILE` still needs to be enabled for kexec support. We only want to disable `CONFIG_KEXEC`.